packages/turf-explode/test.ts

```diff
@@ -11,14 +10,7 @@ const directories = {
   out: path.join(__dirname, "test", "out") + path.sep,
 };
 
-// Save input fixtures
-if (process.env.REGEN) {
-  Object.keys(fixtures).forEach((name) => {
-    writeJsonFileSync(directories.in + name + ".json", fixtures[name]);
-  });
-}
```

Comment:
It only referenced the geojson-fixtures package to regenerate the inputs into the tests, so we already have what we need here.